app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new
+    @like.author = current_user
+    @like.post = Post.find(params[:post_id])
+    if @like.save
+      redirect_to user_post_path(@like.post.author, @like.post)
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] @jorgegoco great work on your likes controller however am unable to add likes to a post, kindly provide a way of adding likes to a post